src/test/java/core/basesyntax/service/strategy/PurchaseHandlerTest.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.service.strategy;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import core.basesyntax.dao.FruitDao;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class PurchaseHandlerTest {
+    @Test
+    void purchaseProcessOperationIs_Ok() {
+        try {
```

Comment:
Please remove all try/catch constructions from tests.
If exception occures during test perform it's logic then it's automatically failed (unless we have assertThows construction).
Even if we have some checked exception in logic we usually just mention it in test method header

```suggestion
```